lib/src/models/usage.dart

```diff
@@ -74,26 +81,20 @@ class Usage {
   /// Total amount of used add-on minutes.
   int? totalFixedMinutesUsed;
 
-  /// Start date of a billing cycle.
-  DateTime? billingCycleStart;
-
-  /// End date of a billing cycle.
-  DateTime? billingCycleEnd;
-
-  /// Indicator if the Explorer has access to a 5 min free call.
-  bool? hasFreeCall;
+  /// Total available minutes in the plan.
+  int? totalMinutes;
 
-  /// Delay before next free 5 min call.
-  DateTime? timeUntilNextFreeCall;
+  /// Total of used minutes in the plan.
+  int? totalMinutesLeft;
 
-  /// Indicates if the User is Primary.
-  bool primary;
+  /// Grand total of minutes used.
+  int? totalMinutesUsed;
```

Comment:
Maybe we should eliminate the grand totals (`totalMinutes`, `totalMinutesLeft`, `totalMinutesUsed`) as they can be calculated from the primary, credit, and fixed values.